components/Cases/CaseLink.jsx

```diff
@@ -6,6 +6,8 @@ const getLink = (recordId, { form_name_overall, ...caseFormData }) => {
     case 'API_Allocation':
     case 'API_Deallocation':
       return `/people/${caseFormData.mosaic_id}/allocations/${caseFormData.allocation_id}?recordId=${recordId}`;
+    case 'ASC_case_note':
+      return `/people/${caseFormData.mosaic_id}/records/${recordId}`;
```

Comment:
I think we should refactor a bit this logic, we need to check that the formName is included in `data/form` as in `CaseRecap`.